cmd/cloud-controller-manager/app/apis/config/v1alpha1/defaults.go

```diff
@@ -21,8 +21,8 @@ import (
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	serviceconfigv1alpha1 "k8s.io/controller-manager/config/v1alpha1"
```

Comment:
This should probably become cmconfigv1alpha1 or ctrlmgrconfigv1alpha1.

Reply:
+1